docs/semgrep-ci/configuration-reference.md

```diff
@@ -69,17 +69,25 @@ To find more details about some of these configuration options, see the followin
 
 ## Diff-aware scanning (`SEMGREP_BASELINE_REF`)
 
-For [diff-aware scans](overview.md#features), this option filters scan results to those introduced after the git commit, in a branch, or tag. For example, you have a repository with 10 commits. You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.
+Semgrep scans can be classified by scope. The scope of a scan refers to what lines of code are scanned in a codebase. When classifying scans by scope, there are two types of scans:
```

Comment:
@adamkvitek This is the only line I've changed since our call since you brought up the ambiguity of "types of scan". Let me know if "scope" is better.

Reply:
LGTM!